app/src/main/java/com/kickstarter/viewmodels/ProjectHolderViewModel.java

```diff
@@ -0,0 +1,497 @@
+
+      this.backingViewGroupIsGone = projectMetadata
+        .map(ProjectUtils.Metadata.BACKING::equals)
+        .map(BooleanUtils::negate);
+
+      this.blurbTextViewText = project.map(Project::blurb);
+      this.categoryTextViewText = project.map(Project::category).filter(ObjectUtils::isNotNull).map(Category::name);
+
+      this.commentsCountTextViewText = project
+        .map(Project::commentsCount)
+        .filter(ObjectUtils::isNotNull)
+        .map(NumberUtils::format);
+
+      this.creatorNameTextViewText = project.map(p -> p.creator().name());
+      this.deadlineCountdownTextViewText = project.map(ProjectUtils::deadlineCountdownValue).map(NumberUtils::format);
+
+      this.featuredViewGroupIsGone = projectMetadata
+        .map(ProjectUtils.Metadata.CATEGORY_FEATURED::equals)
+        .map(BooleanUtils::negate);
+
+      this.featuredTextViewRootCategory = this.featuredViewGroupIsGone
+        .filter(BooleanUtils::isFalse)
+        .compose(combineLatestPair(project))
+        .map(bp -> bp.second.category())
+        .filter(ObjectUtils::isNotNull)
+        .map(Category::root)
+        .filter(ObjectUtils::isNotNull)
+        .map(Category::name);
+
+      this.goalStringForTextView = project
+        .map(p -> this.ksCurrency.format(p.goal(), p, false, true, RoundingMode.DOWN));
+
+      this.locationTextViewText = project
+        .map(Project::location)
+        .filter(ObjectUtils::isNotNull)
+        .map(Location::displayableName);
+
+      this.percentageFundedProgress = project.map(Project::percentageFunded).map(ProgressBarUtils::progress);
+
+      this.percentageFundedProgressBarIsGone = project
+        .map(p -> p.isSuccessful() || p.isCanceled() || p.isFailed() || p.isSuspended());
+
+      this.playButtonIsGone = project.map(Project::hasVideo).map(BooleanUtils::negate);
+
+      this.pledgedTextViewText = project
+        .map(p -> this.ksCurrency.format(p.pledged(), p, false, true, RoundingMode.DOWN));
+
+      this.potdViewGroupIsGone = projectMetadata
+        .map(ProjectUtils.Metadata.POTD::equals)
+        .map(BooleanUtils::negate);
+
+      this.projectDisclaimerGoalReachedDateTime = project
+        .filter(Project::isFunded)
+        .map(Project::deadline);
+
+      this.projectDisclaimerGoalNotReachedString = project
+        .filter(p -> p.deadline() != null && p.isLive() && !p.isFunded())
+        .map(p -> Pair.create(this.ksCurrency.format(p.goal(), p, true), p.deadline()));
+
+      this.projectDisclaimerTextViewIsGone = project.map(p -> p.deadline() == null || !p.isLive());
+
+      this.projectMetadataViewGroupBackgroundDrawableInt = projectMetadata
+        .filter(ProjectUtils.Metadata.BACKING::equals)
+        .map(__ -> R.drawable.rect_green_grey_stroke);
+
+      this.projectMetadataViewGroupIsGone = projectMetadata
+        .map(m -> m != ProjectUtils.Metadata.POTD
+          && m != ProjectUtils.Metadata.CATEGORY_FEATURED
+          && m != ProjectUtils.Metadata.BACKING
+        );
```

Comment:
can we simplify this like we did in the `ProjectCardHolderViewModel`: 

https://github.com/kickstarter/android-oss/blob/master/app/src/main/java/com/kickstarter/viewmodels/ProjectCardHolderViewModel.java#L118

that might be true for some other outputs in this file - it seems like a lot of them are shared.  idk if there's any way to consolidate that logic across the two view models or if it would be over abstraction...

Reply:
oooo i think that line is now pointed at a diff place cause #181 merged, but agreed that there is a bunch of repeated project logic here as in the project card :/